app/model/sdl/Abstract/AppModel.js

```diff
@@ -822,6 +888,14 @@ SDL.ABSAppModel = Em.Object.extend(
         return null;
       };
 
+      this.inactiveWindows.forEach(element => {
+        let windowID = getWindowIDToApplyShow(params, element);
+        if(windowID !== null) {
```

Comment:
```suggestion
        if(windowID !== null) {
```
optional:
using `!=` here would also catch undefined, I am never sure about using type compares for null check.
`if (windowID)` is also a safe way to go